packages/core/src/claim/Claim.ts

```diff
@@ -151,8 +156,16 @@ export default class Claim implements IClaim {
    * @returns A new [[Claim]] object.
    */
 
-  public static decompress(compressedClaim: CompressedClaim): Claim {
-    const decompressedClaim = ClaimUtils.decompress(compressedClaim)
+  public static decompress(
+    compressedClaim: CompressedClaim
```

Comment:
to be nitpicky, the CompressedPartialClaim is missing on the signature

Reply:
What do you mean the signature?

Reply:
could have written input type, but what I meant was the [function signature](https://developer.mozilla.org/en-US/docs/Glossary/Signature/Function)

Reply:
Ok, good to know the specific name. 

Thanks, I have updated it.